modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/APIM574ChangeTheStatusOfAnAPIToPrototypedThroughThePublisherRestAPITestCase.java

```diff
@@ -67,24 +63,21 @@
     public static Object[][] userModeDataProvider() {
         return new Object[][]{
                 new Object[]{TestUserMode.SUPER_TENANT_ADMIN},
-//                new Object[]{TestUserMode.TENANT_ADMIN},
+                new Object[]{TestUserMode.TENANT_ADMIN},
         };
     }
 
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
         super.init(userMode);
-
-        String apiProductionEndpointPostfixUrl = "jaxrs_basic/services/customers/" +
-                "customerservice/customers/123";
-
-        String publisherURLHttp = publisherUrls.getWebAppURLHttp();
-
-        apiPublisher = new APIPublisherRestClient(publisherURLHttp);
-        apiPublisher.login(publisherContext.getContextTenant().getContextUser().getUserName(),
-                publisherContext.getContextTenant().getContextUser().getPassword());
-
-        apiEndPointUrl = gatewayUrlsWrk.getWebAppURLHttp() + apiProductionEndpointPostfixUrl;
+        String gatewayUrl;
+        if (gatewayContextWrk.getContextTenant().getDomain().equals("carbon.super")) {
```

Comment:
We can use the constant - `MultitenantConstants.SUPER_TENANT_DOMAIN_NAME` here.

Reply:
fixed